src/include/script.h

```diff
@@ -92,12 +92,24 @@ extern int LuaCall(lua_State *L, int narg, int nresults, int base, bool exitOnEr
   lua_error(l); \
  } while (0)
 
-#define LuaCheckArgs(l, args) \
-	do { \
-		if (lua_gettop(l) != args) { \
-			LuaError(l, "incorrect argument"); \
-		} \
-	} while (0)
+static void LuaCheckArgs(lua_State *l, const int args)
```

Comment:
```suggestion
inline void LuaCheckArgs(lua_State *l, const int args)
```
or move definition in cpp. (apply to below function too).

Reply:
I was surprised )
https://stackoverflow.com/questions/22102919/static-vs-inline-for-functions-implemented-in-header-files
Thanks.